reV/config/offshore_config.py

```diff
@@ -51,6 +51,23 @@ def sam_files(self):
         """Get the sam files dict"""
         return self['sam_files']
 
+    @property
+    def nrwal_configs(self):
+        """Get the nrwal configs dict"""
+        return self['nrwal_configs']
+
+    @property
+    def offshore_meta_cols(self):
```

Comment:
Is this now going to be values from the offshore/NRWAL inputs?  Maybe re-name it NRWAL_input_cols?

Reply:
yes its offshore data inputs that get piped to the output meta data. I named it to imply that its meta data for the offshore sites that needs to be included in the output meta data. I think the name is okay since it ties this to the meta dataframe?

Reply:
Sounds good to me, maybe just clarify in teh docstring, something along the lines of:
"columns from offshore/NRWAL input data to be passed through to the resource meta data table"

Reply:
for sure, the docstring for the offshore module is much better, i see now that I slacked on the config and cli docstring! Will edit.

Reply:
LOL, no worries man, I've been more than guilty of that ;-)